packages/oui-field/src/field.html

```diff
@@ -23,22 +23,10 @@
         ng-if="$ctrl.isErrorVisible()">
         <span ng-messages="$ctrl.getFirstError()" role="alert">
```

Comment:
I don't understand very well what the meaning of this test :-/

Reply:
What test ?

Reply:
Sorry, I mean the method `getFirstError`

Reply:
Only display the first error message I guess